ui/src/components/TopBar.vue

```diff
@@ -27,7 +27,7 @@
           <v-col cols="5">
             <v-row justify="end" align="center">
             <v-btn class="mr-2"  text >Status: {{ status.status }}</v-btn>
-            <v-btn class="mr-2"  outlined v-if="isAuthenticated" v-bind:href="swagger_link">Swagger API</v-btn>
+            <v-btn class="mr-2"  outlined v-if="isAuthenticated" v-bind:to="swagger_link">Swagger API</v-btn>
```

Comment:
Router links are `to`, not `href`, and this is now handled by the router so it should change